notebook/templates/notebook.html

```diff
@@ -98,7 +98,7 @@
                         <li id="save_checkpoint"><a href="#">{% trans %}Save and Checkpoint{% endtrans %}</a></li>
                         <!-- <hr/> -->
                         <li class="divider"></li>
-                        <li id="restore_checkpoint" class="dropdown-submenu"><a href="#">{% trans %}Revert to Checkpoint{% endtrans %}</a>
+                        <li id="restore_checkpoint" class="dropdown-submenu"><a href="#">{% trans %}Revert to Checkpoint{% endtrans %}<span class="sr-only">Toggle Dropdown</span></a>
```

Comment:
Should these have `{% trans %}` markers to allow them to be translated as well?

Reply:
Added trans tags